packages/grid/x-data-grid-pro/src/tests/detailPanel.DataGridPro.test.tsx

```diff
@@ -493,18 +503,22 @@ describe('<DataGridPro /> - Detail panel', () => {
     expect(getRow(0)).toHaveComputedStyle({ marginBottom: '502px' }); // 500px + 2px spacing
   });
 
-  it('should not reuse detail panel components', () => {
-    function DetailPanel() {
-      const [today] = React.useState(() => new Date());
-      return <div>Date is {today.toISOString()}</div>;
-    }
-    const { setProps } = render(
-      <TestCase getDetailPanelContent={() => <DetailPanel />} detailPanelExpandedRowIds={[0]} />,
-    );
-    expect(screen.queryByText('Date is 1970-01-01T00:00:00.000Z')).not.to.equal(null);
-    clock.tick(100);
-    setProps({ detailPanelExpandedRowIds: [1] });
-    expect(screen.queryByText('Date is 1970-01-01T00:00:00.100Z')).not.to.equal(null);
+  describe('-', () => {
```

Comment:
I did this initially, but then changed it to avoid a weird test name:
`<DataGridPro /> - Detail panel with fake timers should not reuse detail panel components`

But if it's more clear, I'm OK with it

Reply:
IMHO, it's more clear both when reading this whole test name as well as looking at the code. 🤔 
But in both cases, it's just a nitpick as you can easily click on the error line in IDE/console. 😆

Reply:
I rewrote the test to not use fake timers at all.